components/climate/thermostat.rst

```diff
@@ -315,6 +315,93 @@ Set Point Options/Behavior
 **If configured, at least one of** ``default_target_temperature_low`` **and** ``default_target_temperature_high``
 **must be specified in the away mode configuration.**
 
+Presets
+*******
+
+Presets allow you to preconfigure different modes for your thermostat. These will show up in the Home Assistant
+UI where they can change temperature set points, fan modes, fan swing modes, and operating modes all with one
+operation.
+
+- **preset**: (*Optional*, list)
+
+  - **name** (*Required*, string): Name of the preset. If this is one of the *standard* presets (``eco``, ``away``, 
+    ``boost``, ``comfort``, ``home``, ``sleep``, or ``activity``) it is considered a *standard_*preset. Any other
+    string will make the preset a *custom* preset. *Standard* and *custom* presets are functionally equivalent
+    the only difference is that when switching the mode via :ref:`climate.control Action <climate-control_action>`
+    you will need to use the `preset` or `custom_preset` property as appropriate. The Home Assistant 
+    `climate.set_preset_mode` service treats them identically
+  - **default_target_temperature_low** (*Optional*, float): The default low target temperature when switching to 
+    this preset
+  - **default_target_temperature_high** (*Optional*, float): The default high target temperature when switching
+    to this preset.
+  - **mode** (*Optional*, climate mode): The default mode to switch the thermostat when switching to this preset
+    If not specified it will remain on the mode it was on prior to switching. One of:
```

Comment:
If not specified, the thermostat's mode will remain unchanged when the preset is activated. One of: